src/views/GovernanceTaskList/index.tsx

```diff
@@ -29,43 +29,44 @@ import TaskListItem from './TaskListItem';
 
 import './index.scss';
 
-const intakeLinkComponent = (
-  intakeStatus: string,
-  systemIntake: SystemIntakeForm
-) => {
-  const path = chooseIntakePath(systemIntake, intakeStatus);
-  switch (intakeStatus) {
-    case 'COMPLETED':
-      return (
-        <UswdsLink variant="unstyled" asCustom={Link} to={path}>
-          View Submitted Request Form
-        </UswdsLink>
-      );
-    case 'CONTINUE':
+const IntakeLink = ({ intake }: { intake: SystemIntakeForm }) => {
+  switch (intake.status) {
+    case 'INTAKE_SUBMITTED':
       return (
         <UswdsLink
-          className="usa-button"
           variant="unstyled"
           asCustom={Link}
-          to={path}
+          to={`/system/${intake.id}/view`}
         >
-          Continue
+          View Submitted Request Form
         </UswdsLink>
       );
-    case 'START':
+    case 'INTAKE_DRAFT':
+      if (isIntakeStarted(intake)) {
+        return (
+          <UswdsLink
+            className="usa-button"
+            variant="unstyled"
+            asCustom={Link}
+            to={`/system/${intake.id}/contact-details`}
+          >
+            Continue
+          </UswdsLink>
+        );
+      }
       return (
         <UswdsLink
           data-testid="intake-start-btn"
           className="usa-button"
           variant="unstyled"
           asCustom={Link}
-          to={path}
+          to={`/system/${intake.id || 'new'}/contact-details`}
         >
           Start
         </UswdsLink>
       );
     default:
-      return null;
+      return <></>;
```

Comment:
This is an interesting bit that I'm still not 100% certain of. Using `return null` causes some weird Typescript/ESLint errors for React components. I think that's why Kara had this as a Javascript function rather than a component.

Returning an empty partial is new and strange to me, but it's harmless.